File: CS/ChartsDemo.Wpf/ViewModels/CountryPopulationData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace ChartsDemo {
    [XmlRoot("CountryPopulationData")]
    public class CountryPopulationData : List<CountryPopulation> {
        static List<CountryPopulation> data = null;

        public static List<CountryPopulation> Data {
            get {
                if(data == null) {
                    var s = new XmlSerializer(typeof(CountryPopulationData));
                    data = (List<CountryPopulation>)s.Deserialize(DataLoader.LoadFromResources("/Data/CountryPopulationData.xml"));
                }
                return data;
            }
        }
    }
    public class CountryPopulation {
        public string Country { get; set; }
        public int Population { get; set; }
    }
}

File: VB/ChartsDemo.Wpf/ViewModels/CountryPopulationData.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Xml.Serialization

Namespace ChartsDemo
    <XmlRoot("CountryPopulationData")>
    Public Class CountryPopulationData
        Inherits List(Of CountryPopulation)


        Private Shared data_Renamed As List(Of CountryPopulation) = Nothing

        Public Shared ReadOnly Property Data() As List(Of CountryPopulation)
            Get
                If data_Renamed Is Nothing Then
                    Dim s = New XmlSerializer(GetType(CountryPopulationData))
                    data_Renamed = DirectCast(s.Deserialize(DataLoader.LoadFromResources("/Data/CountryPopulationData.xml")), List(Of CountryPopulation))
                End If
                Return data_Renamed
            End Get
        End Property
    End Class
    Public Class CountryPopulation
        Public Property Country() As String
        Public Property Population() As Integer
    End Class
End Namespace
